tox.ini

```diff
@@ -1,39 +1,27 @@
 [tox]
-envlist = py27,py36,static,pidiff
+envlist = py27
 
 [testenv]
-deps=-rtest-requirements.txt
-commands=pytest -v {posargs}
-whitelist_externals=sh
+deps = -rtest-requirements.txt
+commands = pytest -v {posargs}
+whitelist_externals = sh
 
-[testenv:py27]
-deps=
-  # Note: we need to explicitly list requirements.txt here
-  # so it's processed at the same time as the constraints file
-  -rrequirements.txt
-  -rtest-requirements.txt
+[testenv:py26]
+deps = -rrequirements.txt
 
-[testenv:static]
-deps=
-  pylint
-commands=
-  sh -c 'pylint-2 alt-src; test $(( $? & (1|2|4|32) )) = 0'
+[testenv:py27]
+deps = -rrequirements.txt
 
 [testenv:cov]
-deps=
-  -rtest-requirements.txt
-  pytest-cov
-usedevelop=true
-commands=
-  pytest --cov-report=html --cov=pubtools {posargs}
+basepython = python2.7
+usedevelop = true
+commands =
+    pytest --cov-report=html --cov=tests/test_import/alt_src.py {posargs}
```

Comment:
This --cov argument doesn't look correct as it would only measure coverage of that one file, should it rather be --cov=alt_src (note: that may depend on https://github.com/release-engineering/alt-src/pull/2).

Reply:
Well, yeah, from what this PR is based on top of, the only thing *to* show coverage for is the alt-src script (which is converted to the Python module at tests/test_import/alt_src.py).

The previous passed just "alt-src" and with that we end up getting coverage for all site packages in .tox as well. Now, as an alternative to this I can add .tox to the .coveragerc file, which, now that I think about it, is probably preferable.